modules/ossm-about-kiali-otel.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/kiali/ossm-kiali-assembly.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ossm-about-kiali-otel_{context}"]
+= About {DTProductName} and Kiali
```

Comment:
I'd recommend renaming this to something that describes the content better, maybe "Integrating {DTProductName} with {KialiProduct}"?

I'm also not sure re the consistency of using attributes for these product names - it seems like `{KialiProduct}` is sometimes used and sometimes not?

Reply:
{KialiProduct} refers specifically to Kiali Operator provided by Red Hat, which is separate from the Kiali Server, which is separate from the OpenShift Service Mesh Console (OSSMC) plugin, but all are part of Kiali.

If I always use {KialiProduct} then it only references Kiali Operator provided by Red Hat, which will make everything, including titles, extremely long. But if that is what has to happen, then that is what has to happen.